sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientTests.cs

```diff
@@ -76,5 +78,54 @@ public async Task QueryBatchHandledInvalidResponse()
             LogsBatchQueryResult batchResult = await client.QueryBatchAsync(batch);
             Assert.NotNull(batchResult.GetResult("0"));
         }
+
+        [Test]
+        public async Task UsesDefaultEndpoint()
+        {
+            string uri = null;
+            var mockTransport = MockTransport.FromMessageCallback(message =>
+            {
+                uri = message.Request.Uri.ToString();
+                var mockResponse = new MockResponse(200);
+                mockResponse.SetContent("{\"tables\":[]}");
+                return mockResponse;
+            });
+
+            var client = new LogsQueryClient(new MockCredential(), new LogsQueryClientOptions()
+            {
+                Transport = mockTransport
+            });
+
+            await client.QueryAsync("", "", DateTimeRange.All);
+            StringAssert.StartsWith("https://api.loganalytics.io", uri);
+        }
+
+        [TestCase(null, "https://api.loganalytics.io//.default")]
+        [TestCase("https://api.loganalytics.gov//.default", "https://api.loganalytics.gov//.default")]
+        public async Task UsesDefaultAuthScope(string scope, string expectedScope)
+        {
+            var mockTransport = MockTransport.FromMessageCallback(message =>
+            {
+                var mockResponse = new MockResponse(200);
+                mockResponse.SetContent("{\"tables\":[]}");
+                return mockResponse;
+            });
+
+            Mock<MockCredential> mock = new() { CallBase = true };
```

Comment:
FYI - [this change](https://github.com/Azure/azure-sdk-for-net/blob/59f2c7dae9202df896c0e7077d9d154198a88260/sdk/core/Azure.Core.TestFramework/src/MockCredential.cs) (yet to merge) makes it easy to setup callbacks for calls to GetToken.